exporter/awsemfexporter/grouped_metric.go

```diff
@@ -19,6 +19,8 @@ import (
 	"encoding/json"
 	"strings"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cwlogs"
```

Comment:
This import should be after the rest.

Reply:
I ran goimports, Ill give it another go but all lint checks passed